controllers/grid/files/final/FinalDraftFilesGridHandler.inc.php

```diff
@@ -47,21 +46,14 @@ function FinalDraftFilesGridHandler() {
 	// Public handler methods
 	//
 	/**
-	 * Show the form to allow the user to select review files
-	 * (bring in/take out files from submission stage to review stage)
-	 *
-	 * FIXME: Move to it's own handler so that it can be re-used among grids.
-	 *
+	 * Show the form to allow the user to select files from previous stages
 	 * @param $args array
 	 * @param $request PKPRequest
 	 * @return JSONMessage JSON object
 	 */
 	function selectFiles($args, $request) {
-		$submission = $this->getSubmission();
-
 		import('lib.pkp.controllers.grid.files.final.form.ManageFinalDraftFilesForm');
-		$manageFinalDraftFilesForm = new ManageFinalDraftFilesForm($submission->getId());
-
+		$manageFinalDraftFilesForm = new ManageFinalDraftFilesForm($this->getSubmission()->getId());
```

Comment:
for cases like this one, that we use the submission just once, can we start using this kind of approach?